nautobot_ssot/api/views.py

```diff
@@ -0,0 +1,32 @@
+"""Views for SSoT APIs."""
+from rest_framework.routers import APIRootView
+
+from nautobot.extras.api.views import NautobotModelViewSet
+
+from nautobot_ssot.api import serializers
+from nautobot_ssot import filters
+from nautobot_ssot import models
+
+
+class SsotRootView(APIRootView):
```

Comment:
```suggestion
class SSotRootView(APIRootView):
```

Nitpick

Reply:
I thought PEP8 says it should be camelcase?

Reply:
It's OK to have multiple consecutive capitals - I'd vote for `SSOTRootView` myself as I personally prefer acronyms to stay capitalized rather than being treated as TitleCase words. (`nautobot.ipam.models.IPAddress`, not `IpAddress`)

Reply:
Alright, works for me.